recipes/pvl/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
Please delete `bld.bat` and `buld.sh` since you are already using the `script` section of `build`.